src/tests/ftest/control/dmg_storage_query.py

```diff
@@ -173,4 +173,4 @@ def test_dmg_storage_query_device_state(self):
             self.get_dmg_output("storage_set_faulty", uuid=dev[0])
 
         # Check that devices are in FAULTY state
```

Comment:
update comment?

Reply:
the devices are in a faulty state, it just displays EVICTED, I guess that kind of confusing